### DISCUSSION TITLE
Potential restarting device

### DISCUSSION THREAD

**Discussion by pfk-beta** [DeviceFarmer/stf#499] [Q&A] [answered]
Hello,

I plan to deploy stf in my little company to manage some of the phones. I have strange scenario, when sometimes phones restarts during model tuning. Does the STF app will reconnect restarted phones to STF server automatically, or should I do it manually?

**Comment by koral--** [comment, marked as answer]
In general they will be reconnected automatically unless cases when eg. after restart USB is in charge-only mode and you need to tap on the connection type dialog.
